### DISCUSSION TITLE
Mealie is still a NO GO

### DISCUSSION THREAD

**Discussion by sovereignalmida** [Radiokot/umbrel-cloudflared#5] [Q&A] [answered]
Been trying to get this too work with Mealie after having failed using a reverse proxy.   Tunnel is created fine but still cannot access.  The docker-compose.yml calls out a proxy port  of 3000 but the app from Umbrel uses port 9925.  I have tried both and no joy. 

https://github.com/getumbrel/umbrel-apps/blob/master/mealie/docker-compose.yml

Page always times out with this in the address bar:  http://mealie.MYDOMAIN.com:2000/?origin=host&app=mealie&path=%2F):2000/?origin=host&app=mealie&path=%2F

**Comment by Radiokot** [comment, marked as answer]
Hello Chris.
For Mealie, you must use port `3000` and hostname `mealie_mealie-frontend_1`. Your tunnel's public hostname config in Cloudflare dashboard must look like this:
![image](https://github.com/Radiokot/umbrel-cloudflared/assets/5675681/ad7ae3ab-3a5f-4bac-bf21-ca6f5a76d11c)
The result:
![image](https://github.com/Radiokot/umbrel-cloudflared/assets/5675681/ae834505-de34-40f2-8441-2c65b704169b)

**Reply by sovereignalmida** [reply]
Perfect and worked flawlessly.  RTFM in detail next time...  My mistake was using my custom host name and not the Umbrel Mealie server name. Thanks for a great app!

**Comment by markbluhm** [comment]
i know this is marked answered but if anyone else searches for it, 
they updated to `mealie_web_1:9000`

**Reply by Radiokot** [reply]
Thank you

**Reply by cashpw** [reply]
This is still the correct port as of the time of this post. I'm not using Umbrel but I found this while searching for why all other services worked but mealie didn't. I needed to set service `Type` to `HTTP` and `URL` to `mealie:9000` (the name of my mealie container is `mealie`, but you'll need to change this if you use a different name).

**Reply by Radiokot** [reply]
Thanks for sharing, @cashpw
